### PR TITLE
add information about free instances

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-468.surge.sh

Updated pages:
- [Creating an instance](https://neo4j-docs-aura-468.surge.sh/aura/auradb/getting-started/create-database)

**Review by fiquick — COMMENTED** [review]
I’ve noticed that we tend to rely on the word ‘only’ quite a bit. It might help to soften the language and make it more welcoming by exploring alternative phrasing. It might make this page feel more user-friendly- as it's going to appear quite early on in the user's Neo4j journey!

File: modules/ROOT/pages/auradb/getting-started/create-database.adoc [hunk 1]
```diff
@@ -20,10 +20,18 @@ To create an *AuraDB Free* instance in Neo4j AuraDB:
...
-======
+
+You can only create **one** AuraDB Free instance per account.
+
+A Free instance is limited to 200,000 nodes and 400,000 relationships.
+
+If you don't perform any write queries for three days, your instance will be **paused**.
+You can resume your paused instance from the console.
+
+A paused instance is **cancelled after 30 days** and after that, you **cannot restore it or recover your data**.
```
[the remark this answers is not in the document]
Comment (fiquick) [reply modules/ROOT/pages/auradb/getting-started/create-database.adoc:31]: Maybe we drop one of the "restore / recover" words actually - that might soften it up enough 

"A paused instance is **cancelled after 30 days** and after that, you **cannot restore your data**."

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-468.surge.sh

Updated pages:
- [Creating an instance](https://neo4j-docs-aura-468.surge.sh/aura/auradb/getting-started/create-database)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-468.surge.sh

Updated pages:
- [Creating an instance](https://neo4j-docs-aura-468.surge.sh/aura/auradb/getting-started/create-database)

**Review by fiquick — APPROVED** [review]

**Pull request merged**